### PR TITLE
added SonarQube script for static code analysis

### PR DESCRIPTION
**Pull request by Josca**

### CODE REVIEW

File: docker-compose.sonar.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: '3'
```
Comment (semtexzv) [inline docker-compose.sonar.yml:1]: Wouldn't it be better to have these in the separate directory ?

Comment (Josca) [reply docker-compose.sonar.yml:1]: I don't mind to have it here. But we can change the location if you want. Where do you place it?

Comment (semtexzv) [reply docker-compose.sonar.yml:1]: I'd rather place all of the functionality into the single directory, maybe `scripts/sonarqube`. This is not something we use daily, so no need to spread it around through the whole codebase.

Comment (Josca) [reply docker-compose.sonar.yml:1]: There is already /dev/sonar directory. So I will place docker-compose file there.

**Review by MichaelMraka — APPROVED** [review]

**Pull request merged**